packages/fulfillment/src/models/shipping-profile.ts

```diff
@@ -37,6 +43,16 @@ export default class ShippingProfile {
   @PrimaryKey({ columnType: "text" })
   id: string
 
+  @Property({ columnType: "text" })
+  @ShippingProfileTypeIndex.MikroORMIndex()
+  name: string
+
+  @Enum({
+    items: () => ShippingProfileType,
+    default: ShippingProfileType.DEFAULT,
+  })
+  type: ShippingProfileType = ShippingProfileType.DEFAULT
```

Comment:
thought: I would leave this entirely open-ended. Might even make sense to consider if the loader makes sense as this is somewhat opiniated logic, but we can leave that in for now.